torch/csrc/api/src/nn/modules/activation.cpp

```diff
@@ -244,5 +244,17 @@ void SoftsignImpl::pretty_print(std::ostream& stream) const {
   stream << "torch::nn::Softsign()";
 }
 
+// ============================================================================
+
+Tensor TanhImpl::forward(const Tensor& input) {
+  return input.tanh();
```

Comment:
nit: the following would match the Python version even better:
```suggestion
  return torch::tanh(input);
```